packages/core/system/src/types.ts

```
@@ -2,3 +2,381 @@
+  readonly era: string;
+  /** The year of this date within the era. */
+  readonly year: number;
+  /**
+   * The month number within the year. Note that some calendar systems such as Hebrew
+   * may have a variable number of months per year. Therefore, month numbers may not
+   * always correspond to the same month names in different years.
+   */
+  readonly month: number;
+  /** The day number within the month. */
+  readonly day: number;
+  constructor(year: number, month: number, day: number);
```

Biome (1.9.4)

[error] Don't use the constructor method in interfaces.

Interfaces define a contract, not an implementation. Thus, including a constructoris not appropriate.

(lint/suspicious/noMisleadingInstantiator)